docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -20,24 +20,36 @@ default:: Vault reference
 A binary secret key encoded base64.
 It can be generated using the command `openssl rand -base64 128`.
 
-
-== `protocol`
+== `addresses`
 
 [horizontal]
-type:: string
-default:: `layer2`
+type:: ip list
+default:: null
 
-MetalLB supports https://metallb.universe.tf/concepts/layer2/[Layer2] `layer2` and https://metallb.universe.tf/concepts/bgp/[BGP] `bgp`.
-As layer2 is often used on-premis, this is the default.
+A list of IPs (for example `10.128.1.200/32`) used as transferable virtual IPs.
 
 
-== `addresses`
+== `config`
 
 [horizontal]
-type:: ip list
+type:: yaml config
```

Comment:
so technically the value of `config` would be a dictionary, correct?

We'll need to standardize on terminology for component docs at some point

Reply:
Changed to `dictionary`. Even if is not the term used in the future, but at least more accurate than mine.